app/controllers/confirmation_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class ConfirmationController < ApplicationController
+  rescue_from ActiveRecord::RecordNotUnique, with: :error_not_unique
+
+  # This populates instance variables used by the new confirmation form.
+  def new
+    @term = Term.find(params[:term_id])
+    @categories = Category.all
+    @confirmation = Confirmation.new
+  end
+
+  # This receives the submission from the new confirmation form, creating the needed record with the help of various
+  # private methods.
+  def create
+    confirmation = Confirmation.new({
+                                      term_id: params[:term_id],
+                                      user: current_user
+                                    })
+    set_category(confirmation, params[:confirmation][:category])
+    feedback_for(confirmation.save)
+    redirect_to terms_unconfirmed_path
+  end
+
+  private
+
+  # This method takes the result of the confirmation.save directive above and sets an appropriate flash message.
+  #
+  # The final else clause is likely to be difficult to provoke, but I have trouble building an if conditional without
+  # either a default value or an else clause.
+  def feedback_for(result)
+    if result == true && params[:confirmation][:category] == 'flag'
+      flash[:success] = 'Term flagged for review'
+    elsif result == true
+      flash[:success] = "Term confirmed as #{Category.find_by(id: params[:confirmation][:category]).name}"
+    else
+      flash[:error] = 'Unable to finish confirming this term. Please try again, or try a different term.'
```

Comment:
If this is an unexpected state to be in it feels worth throwing an exception to Sentry so we are alerted to it. If this is frequent but we rescue like this and nobody tells us but we thought it could never happen, that could be difficult to detect without a mechanism to inform us.

Something like this should work:

`Sentry.capture_message 'Unexpected error confirming term: #{some maybe useful id of the problem term?}'`

More info in the docs:
https://docs.sentry.io/platforms/ruby/guides/rails/usage/#capturing-messages

Reply:
I was tempted to send something to Sentry, both for this message as well as the RecordNotUnique catch, but then noticed that we don't have it set up. I wasn't sure if that should be a side effect to this work, or if it should be something that gets specific focus?

Reply:
Ah... we do have a ticket for that. I forgot. Hmmm. I'd be okay with opening a ticket to add the Sentry logging later (or puling that Sentry ticket and putting in in place before merging this).

Reply:
Given that we both thought about leveraging Sentry here, I think it's appropriate to pull TCO-88 now, get that done, and then update this.

Reply:
Now that #146 has merged, I'm going to rebase this branch and add the Sentry message here.